doc/api/perf_hooks.md

```diff
@@ -1306,6 +1306,24 @@ http.createServer((req, res) => {
 });
 ```
 
+### Measuring how long fs sync API takes when the operation is successful
+
+```js
+'use strict';
+const { PerformanceObserver } = require('perf_hooks');
+const fs = require('fs');
+
+const obs = new PerformanceObserver((items) => {
+  items.getEntries().forEach((item) => {
+    console.log(item);
+  });
+});
+
+obs.observe({ entryTypes: ['fs'] });
```

Comment:
Since this is covering sync fs specifically, perhaps the entry name should be `syncfs` or `sync-fs`, etc

Reply:
> Since this is covering sync fs specifically, perhaps the entry name should be `syncfs` or `sync-fs`, etc

i think we maybe trace other api or things of fs?